### PR TITLE
Add global setting for page preference for second tab when starting app

### PR DESCRIPTION
**Pull request by tmccormi**

### CODE REVIEW

File: library/globals.inc.php [hunk 1]
```diff
@@ -5,65 +5,16 @@
...
-//   Serbian                        // xl('Serbian')
-//   Sinhala                        // xl('Sinhala')
-//   Slovak                         // xl('Slovak')
-//   Somali                         // xl('Somali')
-//   Spanish (Latin American)       // xl('Spanish (Latin American)')
-//   Spanish (Spain)                // xl('Spanish (Spain)')
-//   Swedish                        // xl('Swedish')
-//   Turkish                        // xl('Turkish')
-//   Ukrainian                      // xl('Ukrainian')
-//   Vietnamese                     // xl('Vietnamese')
```
Comment (teryhill) [inline library/globals.inc.php]: You know these are here to translate the languages

File: library/globals.inc.php [hunk 2]
```diff
@@ -82,34 +33,15 @@
...
-// xl('Security')
-// xl('Notifications')
-// xl('CDR')
-// xl('Logging')
-// xl('Miscellaneous')
-// xl('Portal')
-// xl('Connectors')
-// xl('RX')
-// xl('PDF')
-
```
Comment (teryhill) [inline library/globals.inc.php]: The same with these , to allow translation of the tabs

**Comment by tmccormi** [comment]
They are just comments?  why would the be needed?  I can, of course put them back if some external process using them, but geez.

**Comment by teryhill** [comment]
Translation routines look for the xl where ever they are

**Comment by teryhill** [comment]
Don't see any obvious problems in the code, does it test well?

**Comment by tmccormi** [comment]
Just found a bug.  It's not letting the tab 1 get changed.  it's hardcoded to calendar somewhere.  gonna ping Kevin

**Comment by tmccormi** [comment]
interesting. It works as a global setting, but not as a user_pref setting

**Comment by teryhill** [comment]
what is in the globals table for the records

**Comment by tmccormi** [comment]
doesn't appear to be writing to the globals table at all for those defaults.  Do you see anything obvious in the globals.inc.php that I'm blind too?

**Comment by tmccormi** [comment]
weird,  it doesn't save default_tab_2 in user settings and it ignores default_tab_1, but not always...
user_settings

**Comment by teryhill** [comment]
just for kicks change the name of the globals to first_default_tab and second_default_tab

**Comment by tmccormi** [comment]
what good is that, the code uses the current names...  I don't understand what you are trying to see.

**Comment by teryhill** [comment]
if it was being truncated some where but since you see them now scratch that

**Comment by tmccormi** [comment]
I have new code to test and update ... please hold

**Comment by aethelwulffe** [comment]
Tell me when it is at test stage.
  I have a lot of catching up to do

**Comment by tmccormi** [comment]
see issue #355 I'm testing my branch merge now and will push when I have decent results

**Comment by tmccormi** [comment]
This is now testing well on my side, and ready for other to poke at... @aethelwulffe @teryhill

**Comment by aethelwulffe** [comment]
OK,

Seemed to work.  I get billing report now.

**Comment by tmccormi** [comment]
Two tabs seems good to me.  Now that I know how yehster did it we could add a third.   Better would be to have a 'save last' function for users (like demographics).  Then the user would be able to override even the defaults ... future.

**Pull request merged**